src/Component.php

```diff
@@ -362,7 +362,11 @@ public function executeAjaxCallback(\PaymentMethod $method, &$form, &$form_state
     }
     $payment = $this->payment;
     $payment->method = $method;
-    $result = $method->controller->ajaxCallback($payment, $form, $form_state);
+    // Pass only controller specific form data to the callback.
```

Comment:
This comment is only helpful when you take into account that the whole form element was passed in the commmit before. As a description for the current behaviour I find it rather confusing. I would either leave it or change it to something like:

> Pass method form element to the callback.

Reply:
I'm taking into account that the whole $form (presumably) was passed as argument and now we filter out the relevant part. But happy to remove the comment entirely :)

Reply:
Yes, the whole `$form` was passed — in your the previous commit if this very PR. This branch introduces two new arguments.